src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -136,7 +136,7 @@
             </trans-unit>
             <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
-                <target state="needs-review-translation">Šī vērtība nav derīga IP adrese.</target>
+                <target>Šī nav derīga IP adrese.</target>
```

Comment:
you're removing `vērtība` but we want the translation to be as if the english version where "This *value* is not a valid IP address." (same for other updates). Should we keep the word?

Reply:
Or use `Šai vērtībai`  as in other existing messages that start with "This value is not"?

Reply:
but source part now does not contain word "value". "vērtība" literally means "value"

Reply:
yes, that's exaclty what I mean.
See #53507